packages/reports/addon/consumers/request/rollup.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2021, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+import { inject as service } from '@ember/service';
+import ActionConsumer from 'navi-core/consumers/action-consumer';
+import RequestActionDispatcher, { RequestActions } from 'navi-reports/services/request-action-dispatcher';
+import type Route from '@ember/routing/route';
+import type ColumnFragment from 'navi-core/models/bard-request-v2/fragments/column';
+import type ReportModel from 'navi-core/models/report';
+
+export default class RollupConsumer extends ActionConsumer {
+  @service
+  declare requestActionDispatcher: RequestActionDispatcher;
+
+  actions = {
```

Comment:
How will handle reorder? Will that come in another PR?

Reply:
the order of columns does not effect rollups. it's always the order the person pushes the rollup buttons.

Reply:
I believe we decided that behavior was sufficient for the alpha branch, but wanted to utilize the column reorder feature for rollup ordering in beta

Reply:
We are only adding behavior to equal to alpha branch for now.